Classes/Items.gd

```diff
@@ -0,0 +1,141 @@
+class_name Items
+const script_name := "items"
+
+const items := {
+	
+	
+	
+	
+	sword = {
+		nickname = "Sword",
+		name = "sword",
+		type = "melee",
+		hand = "one-handed",
+		tags = [],
+		buffs = {
+			strength = 0,
+			perception = 0,
+			endurance = 0,
+			charisma = 0,
+			intelligence = 0,
+			agility = 0,
+			luck = 0,
+		},
+		level = 1,
+		level_requirement = 1,
+	},
+	
+	
+	
+	
+	staff = {
+		nickname = "Staff",
+		name = "staff",
+		type = "magic",
+		hand = "one-handed",
+		tags = [],
+		buffs = {
+			strength = 0,
+			perception = 0,
+			endurance = 0,
+			charisma = 0,
+			intelligence = 0,
+			agility = 0,
+			luck = 0,
+		},
+		level = 1,
+		level_requirement = 1,
+	},
+	
+	
+	
+	
+	axe = {
+		nickname = "Battle Axe",
+		name = "axe",
+		type = "melee",
+		hand = "two-handed",
+		tags = [],
+		buffs = {
+			strength = 0,
+			perception = 0,
+			endurance = 0,
+			charisma = 0,
+			intelligence = 0,
+			agility = 0,
+			luck = 0,
+		},
+		level = 1,
+		level_requirement = 1,
+	},
+	
+	
+	
+	
+	bow = {
+		nickname = "Hunting bow",
+		name = "bow",
+		type = "ranged",
+		hand = "two-handed",
+		tags = [],
+		buffs = {
+			strength = 0,
+			perception = 0,
+			endurance = 0,
+			charisma = 0,
+			intelligence = 0,
+			agility = 0,
+			luck = 0,
+		},
+		level = 1,
+		level_requirement = 1,
+	},
+	
+	
+	
+	
+	arrow = {
+		nickname = "Arrow",
+		name = "arrow",
+		type = "consumables",
+		hand = "one-handed",
+		tags = [],
+		buffs = {
+			strength = 0,
+			perception = 0,
+			endurance = 0,
+			charisma = 0,
+			intelligence = 0,
+			agility = 0,
+			luck = 0,
+		},
+		level = 1,
+		level_requirement = 1,
+	},
+	
+	
+	
+	
+	dagger = {
+		nickname = "Dagger",
+		name = "dagger",
+		type = "melee",
+		hand = "one-handed",
+		tags = [],
+		buffs = {
+			strength = 0,
+			perception = 0,
+			endurance = 0,
+			charisma = 0,
+			intelligence = 0,
+			agility = 0,
+			luck = 0,
+		},
+		level = 1,
+		level_requirement = 1,
+	}
```

Comment:
While this is ok, because these are called each time a character is created, not all weapons will be hard coded. Some weapons and items will be dynamically created and leveled.